api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -310,14 +310,19 @@ private Runtime getOverrideFromListRuntimes(String googleProject) {
   }
 
   @Override
-  public ResponseEntity<EmptyResponse> deleteRuntime(String workspaceNamespace) {
+  public ResponseEntity<EmptyResponse> deleteRuntime(
+      String workspaceNamespace, Boolean deleteDisk) {
+    if (deleteDisk == null) {
```

Comment:
I think this will break UI if you did not set that in UI. Because UI is passing null now:
https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/runtime-utils.tsx#L303

My previous comment is suggesting use `false` if the value is null request. Like:
`deleteDisk == null? false: deletDisk`

Or you can change UI along with the same PR, but I am note sure how many places you need to change, it would be just that line.

Reply:
That makes sense! I prefer the first option. It's more complete for the backend api itself.